runtime/kusama/src/xcm_config.rs

```diff
@@ -401,6 +416,11 @@ impl pallet_xcm::Config for Runtime {
 	// transfer.
 	type XcmReserveTransferFilter = Everything;
 	type Weigher = FixedWeightBounds<BaseXcmWeight, RuntimeCall, MaxInstructions>;
```

Comment:
Doesn't Kusama use `WeightInfoBounds` as a weigher? Maybe you need to rebase or something?